server.js

```diff
@@ -14,6 +14,9 @@ const send = require("./server/routes/send");
 // Passport Config
 require("./server/config/passport")(passport);
 
+// Associations
+const associations = require('./server/routes/associations');
```

Comment:
Comment this out. This might give an error but definitely a warning.

Reply:
Commented